app/models/manageiq/providers/redhat/infra_manager/vm.rb

```diff
@@ -47,8 +47,4 @@ def self.calculate_power_state(raw_power_state)
   def validate_publish
     validate_unsupported("Publish VM")
   end
-
-  def validate_clone
```

Comment:
because clone seems not to be supported by ovirt you have to make it unsupported here:

adding this to the top is enough
```ruby
supports_not :clone
```